packages/node_modules/glimmer-syntax/lib/parser/handlebars-node-visitors.ts

```diff
@@ -107,7 +107,11 @@ export default {
       changeLines = leadingNewlineDifference(content.original, content.value);
     }
 
-    this.tokenizer.line = this.tokenizer.line + changeLines;
+    if (changeLines) {
```

Comment:
Actually on further reflection I see that the variable was used in the deleted line. TLDR move the line and the column reset into the original conditional.